apps/cowswap-frontend/src/legacy/utils/trade.ts

```diff
@@ -290,16 +299,24 @@ export async function sendOrderCancellation(params: OrderCancellationParams): Pr
 
   if (!signature) throw new Error('Signature is undefined!')
 
-  await orderBookApi.sendSignedOrderCancellations(
-    {
-      orderUids: [orderId],
-      signature,
-      signingScheme,
-    },
-    { chainId }
-  )
-
-  cancelPendingOrder({ chainId, id: orderId })
+  try {
```

Comment:
The catch block seems identical for `sendOrder` and `sendSignedOrderCancellations` calls. Would be great to create a function to get rid of the code repetition

Reply:
Good idea 02e0a3198a3168f24cd104a796a3dbcbc3fdd59b